### PR TITLE
API Changes for getting API Document Template Settings Change

### PR DESCRIPTION
**Pull request by TristanAbarca**
Added API Call for getting Document Template Settings.

### CODE REVIEW

File: packages/quip-apps-api/src/client.ts [hunk 1]
```diff
@@ -165,6 +166,13 @@ export default class Client {
     public dateToPickValue: number = new Date().getTime();
     public displayWidthValue: number = 800;
     public documentMembersValue: User[] = [];
+    public documentTemplateSettingsValue: {
```
Comment (XinchenApril) [inline packages/quip-apps-api/src/client.ts:169]: can you define 
```
type DocumentTemplateSettings = {
    salesforce_org_id: string;
    salesforce_object_type: string;
};
```
somewhere, and then here you can do public documentTemplateSettingsValue: DocumentTemplateSettings = {
        salesforce_org_id: "",
        salesforce_object_type: ""
    }
instead?

Comment (TristanAbarca) [reply packages/quip-apps-api/src/client.ts:169]: I could but I was following the same syntax as some of the other types. Do you still think I should make these changes?

File: packages/quip-apps-api/src/client.ts [hunk 2]
```diff
@@ -165,6 +166,13 @@ export default class Client {
...
+        salesforce_org_id: string;
+        salesforce_object_typ: string;
```
Comment (XinchenApril) [inline packages/quip-apps-api/src/client.ts]: typ -> type

File: packages/quip-apps-api/src/client.ts (hunk 1 above)
Comment (XinchenApril) [reply packages/quip-apps-api/src/client.ts:169]: let's keep it

**Review by XinchenApril — APPROVED** [review]

**Pull request merged**